### PR TITLE
 Add description of Information and Metadata tabs 

### LINKED ISSUE #1823: [needs-docs] update metadata viewer and review UI for vector layer dialog (#4316)
Original commit: https://github.com/qgis/QGIS/commit/c4ad9505fbde595481e822c037ac3b23689f3866 by timlinux

Funded by WorldBank/GFDRR

### LINKED ISSUE #2122: [FEATURE] [needs-docs] Metadata editor
Original commit: https://github.com/qgis/QGIS/commit/ba81068f495423380dde0a7a8f81135cf142865d by nyalldawson

Adds a new metadata tab to layer properties, representing the new metadata model from [QEP #91](https://github.com/qgis/QGIS-Enhancement-Proposals/issues/91)

### PR DESCRIPTION
**Pull request by DelazJ**
Also fix some references to the removed layer properties --> General tab
Fixes #1823 
Fixes #2122

### CODE REVIEW

**Comment by DelazJ** [comment]
@timlinux @Gustry can any of you please give a look to the Information and Metadata tabs description? Not sure if i do not miss any important information. Thanks for your help.

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -2745,6 +2763,37 @@ More information on variables usage in the General Tools
...
 
 
+.. index:: Metadata, Metadata editor, Keyword
+.. _vectormetadatamenu:
+
+Metadata Properties
+===================
+
+|editMetadata| The :guilabel:`Metadata` tab provides you with options to create
+and edit a standards compliant metadata report on your layer. Information to
```
Comment (Gustry) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: "standards compliant metadata"

I don't know what would be the best wording. But in the current state, this metadata editor is focused on the QGIS Metadata Internal schema described here https://github.com/qgis/QGIS/pull/4330

Then according to the QEP 91 https://github.com/qgis/QGIS-Enhancement-Proposals/issues/91 in another round of work in QGIS (3.2 or 3.4? @timlinux ) we would be able to finally export/import this internal QGIS schema to a compliant metadata document (ISO, Dublin Core ...) So for now, this metadata in QGIS 3.0 is mainly internal within QGIS.

Comment (timlinux) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: Yes, the new round of funding would bring standards compliance in the form of ability to read and write ISO metadata. We don't have funders yet so let's not give a specific version.

**Comment by timlinux** [comment]
LGTM thanks very much for this @DelazJ

File: source/docs/user_manual/working_with_vector/vector_properties.rst (hunk 1 above)
Comment (DelazJ) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: Thanks I think I'll just remove the "standards compliant" mention. Not sure it's worth (me) going through the QGIS schema and let's hope funders  will come soon.

**Comment by DelazJ** [comment]
Thanks for your reviews

File: source/docs/user_manual/working_with_vector/vector_properties.rst (hunk 1 above)
Comment (Gustry) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: Yes, I think it's fine like this ;-)

**Pull request merged**